src/devices/Adxl345/Adxl345.cs

```diff
@@ -0,0 +1,146 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Device.Spi;
+using System.Runtime.InteropServices;
+using System.Device.Spi.Drivers;
+
+namespace Iot.Device.Adxl345
+{
+    /// <summary>
+    /// SPI Accelerometer ADX1345
+    /// </summary>
+    public class Adxl345 : IDisposable
+    {
+        private SpiDevice _sensor = null;
+
+        private readonly int _busId;
+        private readonly OSPlatform _os;
+        private readonly int _cs;                                
```

Comment:
not sure what _cs means, any chance you could expand the name? (unless this is a name from the spec or pin name on the board)